private_gpt/settings/settings.py

```diff
@@ -120,6 +120,10 @@ class LLMSettings(BaseModel):
 
 class VectorstoreSettings(BaseModel):
     database: Literal["chroma", "qdrant", "postgres"]
+    inject_win_size: int = Field(
+        3,
+        description="How many sentences on either side to capture, when parsing files",
+    )
```

Comment:
Can you move this property to another setting? It would be nice if we have different providers like graphs, elastic search, etc.